### PR TITLE
Organizations

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

File: modules/ROOT/pages/visual-tour/index.adoc [hunk 1]
```diff
@@ -28,6 +28,17 @@ This opens a dropdown menu where you can see all the organizations and projects
 image::breadcrumbs.png[]
 
 [[org-settings]]
+
+=== About Organizations
+
+Organizations are a layer in the Aura account hierarchy that sits above Projects. 
+This feature allows you to better organize and manage multiple projects under a single organization.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/visual-tour/index.adoc:35]: _Organizations_ is not a feature. Maybe we can say
```suggestion
This allows you to organize and manage multiple projects under a single organization.
```

File: modules/ROOT/pages/visual-tour/index.adoc [hunk 2]
```diff
@@ -28,6 +28,17 @@ This opens a dropdown menu where you can see all the organizations and projects
 image::breadcrumbs.png[]
 
 [[org-settings]]
+
+=== About Organizations
```
Comment (AlexicaWright) [inline modules/ROOT/pages/visual-tour/index.adoc:32]:
```suggestion
=== Organizations
```

File: modules/ROOT/pages/visual-tour/index.adoc [hunk 3]
```diff
@@ -28,6 +28,17 @@ This opens a dropdown menu where you can see all the organizations and projects
...
+
+Organizations are a layer in the Aura account hierarchy that sits above Projects. 
+This feature allows you to better organize and manage multiple projects under a single organization.
+
+* Organizations are useful for SSO, user management.
+* There can be a portfolio of companies using one account. 
+* The ability to rollout those Project level settings in one go, rather than having to configure them one at a time which is a  a time-saver that’s really useful because it lets you make changes at an overarching level, applying them across all projects at once.
+* `Organization owners` are a new type of role.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/visual-tour/index.adoc:40]:
```suggestion
Management at this level allows control over all Projects within an Organization and, thus, instances within Projects and lets Organization owners roll out Project-level settings for all Projects simultaneously.
```

**Review by AlexicaWright — COMMENTED** [review]
A suggestion. Also, I don't think multiple companies use the same Aura account. Where did this info come from?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-537.surge.sh

Updated pages:
- [Visual tour of the console](https://neo4j-docs-aura-537.surge.sh/aura/visual-tour)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-537.surge.sh

Updated pages:
- [Visual tour of the console](https://neo4j-docs-aura-537.surge.sh/aura/visual-tour)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-537.surge.sh

Updated pages:
- [Visual tour of the console](https://neo4j-docs-aura-537.surge.sh/aura/visual-tour)

**Pull request merged**